libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -3009,7 +3009,11 @@ def process_line(self, line):
             print("Adding environment %s" % ' '.join(a[1:]))
             if len(a[1:]) < 2:
                 self.error("Bad env line for %s" % a[0])
-            self.env_vars[a[1]] = ' '.join(a[2:])
+            name = a[1]
+            value = ' '.join(a[2:])
+            if name == 'AP_PERIPH':
```

Comment:
we can make non-1 fatal, but don't change the type, leave as a string for consistency